src/zos_files/zowe/zos_files_for_zowe_sdk/files.py

```diff
@@ -36,10 +36,13 @@ def __init__(self, connection):
         ----------
         connection
             The z/OSMF connection object (generated by the ZoweSDK object)
+
+        Also update header to accept gzip encoded responses
         """
         super().__init__(connection, "/zosmf/restfiles/")
+        self.default_headers["Accept-Encoding"] = "gzip"
```

Comment:
I like the approach of adding the header in a common function so we don't have to do it in multiple places 👍 

Do you know if the Python `requests` library automatically decompresses REST responses that have the `Content-Encoding: gzip` header defined? If it doesn't, there will be more changes required to decode compressed responses.

Reply:
Yes, according to the FAQ, it does https://2.python-requests.org/en/master/community/faq/#encoded-data

However,  this issue on github https://github.com/psf/requests/issues/2446 , and
 https://2.python-requests.org/en/master/user/quickstart/#binary-response-content 
mentions that only those responses are decoded which have headers `Transfer-Encoding: gzip`